plasmapy/atomic/classes.py

```diff
@@ -369,3 +369,31 @@ def spin(self) -> Union[int, float]:
             raise MissingAtomicDataError(
                 f"The spin of particle '{self.particle}' is unavailable.")
         return self._spin
+
+    @property
+    def reduced_mass(self, other) -> u.kg:
+        r"""Finds the reduced mass between two particles."""
+
+        try:
+            mass_this = self.mass
+        except MissingAtomicDataError:
+            raise MissingAtomicDataError(
+                f"Unable to find the reduced mass because the mass of "
+                f"{self.particle} is not available.")
+
+        try:
+            mass_that = other.mass
+        except MissingAtomicDataError:
+            raise MissingAtomicDataError(
+                f"Unable to find the reduced mass because the mass of "
+                f"{other.particle} is not available.")
+        except AttributeError:
+            raise
```

Comment:
I don't know too too much about exception handling, but is there really that much point to this sort of "catch exception, do nothing in particular with it"?